frontend/src/widgets/OverviewWidgetField.scss

```diff
@@ -0,0 +1,51 @@
+@media( min-width: 1024px ){
```

Comment:
There's duplicative CSS at frontend/src/widgets/DashboardOverview.css, if we condense the components we can probably condense the CSS as well

Reply:
This widget will need to be updated to use the new overview widget when we do this it will go away. I have only updated resource and qa dashboard atm.

Reply:
what is the new overview widget?

Reply:
So we have a overview widget container now that wraps around all the individual overview widgets. I think this needs to be updated to that structure.

Reply:
We don't need to pass array of field components just an array of object info.